tests/test_rotate.py

```diff
@@ -1,4 +1,4 @@
-from visumorph import rotate, load_image
+from visumorph import scale, load_image
```

Comment:
`test_rotate` should not be related to `scale` so I'm not too sure why is this change here.

Reply:
thanks, changes made accordingly

Reply:
Resolving this as this change is reverted..